<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenFile</class>
 <widget class="QWidget" name="OpenFile">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open Files</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/folder.png</normaloff>images/folder.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Bitstream Vera Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>SPS:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineSFile">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSFile">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/new/prefix1/image/Nuvola_filesystems_folder_yellow_open.png</normaloff>:/new/prefix1/image/Nuvola_filesystems_folder_yellow_open.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="shortcut">
        <string>F1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Bitstream Vera Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>RPS:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineRFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnRFile">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/new/prefix1/image/Nuvola_filesystems_folder_blue_open.png</normaloff>:/new/prefix1/image/Nuvola_filesystems_folder_blue_open.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Bitstream Vera Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>XPS:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineXFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnXFile">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/new/prefix1/image/Nuvola_filesystems_folder_grey_open.png</normaloff>:/new/prefix1/image/Nuvola_filesystems_folder_grey_open.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupUnit">
     <property name="title">
      <string>Unit width:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Bitstream Vera Sans</family>
           </font>
          </property>
          <property name="text">
           <string>EastWest width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineSingleWid">
          <property name="text">
           <string>50</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Bitstream Vera Sans</family>
           </font>
          </property>
          <property name="text">
           <string>SourthNorth width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineSingleHei">
          <property name="text">
           <string>25</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/new/prefix1/image/3.png</normaloff>:/new/prefix1/image/3.png</iconset>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>84</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/new/prefix1/image/8.png</normaloff>:/new/prefix1/image/8.png</iconset>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnSFile</tabstop>
  <tabstop>btnRFile</tabstop>
  <tabstop>btnXFile</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>lineSFile</tabstop>
  <tabstop>lineRFile</tabstop>
  <tabstop>lineXFile</tabstop>
 </tabstops>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
